### PR TITLE
Feature/jcsz1 fusy 7

### PR DESCRIPTION
**Pull request by prz3m3**
JCSZ1FUSY-3 "Konwersja walut "is waiting for a check.

### CODE REVIEW

File: WalutyBusinessLogic/Conversation/CurrencyConvertion.cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+using WalutyBusinessLogic.LoadingFromFile;
+
+namespace WalutyBusinessLogic
+{
+    class CurrencyConvertion
+    {
+        CurrencyRecord currencyRecord = new CurrencyRecord();
+        CurrencyForConversation firstCurrencyConvertion = new CurrencyForConversation();
+        CurrencyForConversation secondCurrencyConvertion = new CurrencyForConversation();
+
```
Comment (djagiellowicz) [inline WalutyBusinessLogic/Conversation/CurrencyConvertion.cs]: CurrencyForConversation zmieniłbym na CurrencyForConversion albo CurrencyConversion. Conversation to rozmowa

File: WalutyConsoleApp/Program.cs [hunk 2]
```diff
@@ -1,13 +1,12 @@
 ﻿using System;
-
+using WalutyBusinessLogic.CurrencyConversion;
```
Comment (djagiellowicz) [inline WalutyConsoleApp/Program.cs]: Twój konflikt to raczej ten using

**Comment by TDerkacz** [comment]
Hi Przemek, please resolve conflicts first

File: WalutyBusinessLogic/Conversation/CurrencyForConversation.cs [hunk 3]
```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using WalutyBusinessLogic.LoadingFromFile;
+
+namespace WalutyBusinessLogic.CurrencyConversion
+{
+
+    public class CurrencyForConversation
```
Comment (TDerkacz) [inline WalutyBusinessLogic/Conversation/CurrencyForConversation.cs]: wrong class name >>>> CurrencyForConversation

Comment (TDerkacz) [inline WalutyBusinessLogic/Conversation/CurrencyForConversation.cs]: wrong class name >>>> CurrencyForConversation

Comment (TDerkacz) [inline WalutyBusinessLogic/Conversation/CurrencyForConversation.cs]: and file name: CurrencyForConversation.cs

**Comment by TDerkacz** [comment]
All required has been applied.

**Pull request merged**